Sources/Brave/Frontend/Shields/SubmitReportSuccessView.swift

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Strings
+import BraveUI
+import DesignSystem
+import SnapKit
+
+struct SubmitReportSuccessView: View {
+  var body: some View {
+    VStack(alignment: .center, spacing: 16) {
+      Image(braveSystemName: "leo.check.circle-outline")
+        .resizable()
+        .frame(width: 48, height: 48)
+        .foregroundStyle(Color(braveSystemName: .systemfeedbackSuccessIcon))
+      
+      Text(Strings.Shields.siteReportedTitle)
+        .multilineTextAlignment(.center)
```

Comment:
As mentioned before `multilineTextAlignment` can be moved to the containing `VStack`

Reply:
done